README.md

```diff
@@ -40,7 +40,13 @@ $ chmod +x coveralls.phar
 To install php-coveralls with Composer, run the following command:
 
 ```sh
-$ composer require satooshi/php-coveralls --dev
+$ composer require satooshi/php-coveralls
```

Comment:
http://www.lornajane.net/posts/2015/relying-on-a-dev-master-dependency-in-composer